scripts/dev.sh

```diff
@@ -85,4 +85,4 @@ else
 fi
 
 # Run bos-loader with updated replacements
-~/.cargo/bin/bos-loader "$ACCOUNT_ID" --path src -r "$REPLACEMENTS_JSON.tmp"
+~/.cargo/bin/bos-loader "$ACCOUNT_ID" --path dist -r "$REPLACEMENTS_JSON.tmp"
```

Comment:
It would be great if we migrate all these replacements to `build.js` and potentially even avoid using bos-loader replacements in favor of some more natural syntax.

Reply:
I agree, that would make it more straight forward